vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -78,7 +81,9 @@ class TextShadowNode final : public ShadowNodeBase {
 
       m_prevCursorEnd += run.Text().size();
     } else if (auto span = static_cast<ShadowNodeBase &>(child).GetView().try_as<winrt::Span>()) {
-      AddNestedTextHighlighter(m_ColorValue, span, static_cast<VirtualTextShadowNode &>(child).m_highlightData);
+      const auto &virtualTextNode = static_cast<VirtualTextShadowNode &>(child);
+      AddNestedTextHighlighter(m_ColorValue, span, virtualTextNode.m_highlightData);
```

Comment:
fyi, there's a tiny conflict with a PR I merged today - you'll need to also also pass the foregroundColor